eth/test/DFArtifacts.test.ts

```diff
@@ -1,22 +1,23 @@
-import { ArtifactRarity, ArtifactType, Biome } from '@dfdao/types';
-import { loadFixture } from '@nomicfoundation/hardhat-network-helpers';
+import { ArtifactRarity, ArtifactType, Biome, SpaceshipType, TokenType } from '@dfdao/types';
```

Comment:
Just gotta humble brag that now we're actually testing all of the artifacts and spaceships (previously were missing photoid, planetary shield, and crescent)